test-clients/src/main/java/com/hedera/services/bdd/suites/file/DiverseStateValidation.java

```diff
@@ -0,0 +1,156 @@
+import java.math.BigInteger;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicReference;
+
+import static com.hedera.services.bdd.spec.HapiApiSpec.defaultHapiSpec;
+import static com.hedera.services.bdd.spec.assertions.ContractFnResultAsserts.isLiteralResult;
+import static com.hedera.services.bdd.spec.assertions.ContractFnResultAsserts.resultWith;
+import static com.hedera.services.bdd.spec.infrastructure.meta.ContractResources.LUCKY_NO_LOOKUP_ABI;
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.contractCallLocal;
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.getContractBytecode;
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.getFileContents;
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.getFileInfo;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.systemFileUndelete;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.logIt;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.sourcing;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.withOpContext;
+import static com.hedera.services.bdd.suites.file.DiverseStateCreation.ENTITY_NUM_KEY;
+import static com.hedera.services.bdd.suites.file.DiverseStateCreation.EXPECTED_LUCKY_NO;
+import static com.hedera.services.bdd.suites.file.DiverseStateCreation.FUSE_BYTECODE;
+import static com.hedera.services.bdd.suites.file.DiverseStateCreation.FUSE_CONTRACT;
+import static com.hedera.services.bdd.suites.file.DiverseStateCreation.FUSE_INITCODE;
+import static com.hedera.services.bdd.suites.file.DiverseStateCreation.HEXED_BYTECODE_KEY;
+import static com.hedera.services.bdd.suites.file.DiverseStateCreation.KEY_REPRS_KEY;
+import static com.hedera.services.bdd.suites.file.DiverseStateCreation.LARGE_CONTENTS_LOC;
+import static com.hedera.services.bdd.suites.file.DiverseStateCreation.LARGE_FILE;
+import static com.hedera.services.bdd.suites.file.DiverseStateCreation.MEDIUM_FILE;
+import static com.hedera.services.bdd.suites.file.DiverseStateCreation.MULTI_CONTRACT;
+import static com.hedera.services.bdd.suites.file.DiverseStateCreation.MULTI_INITCODE;
+import static com.hedera.services.bdd.suites.file.DiverseStateCreation.SMALL_CONTENTS_LOC;
+import static com.hedera.services.bdd.suites.file.DiverseStateCreation.SMALL_FILE;
+import static com.hedera.services.bdd.suites.file.DiverseStateCreation.STATE_META_JSON_LOC;
+
+public class DiverseStateValidation extends HapiApiSuite {
+	private static final Logger log = LogManager.getLogger(DiverseStateValidation.class);
+
+	private static byte[] SMALL_CONTENTS;
+	private static byte[] LARGE_CONTENTS;
+
+	public static void main(String... args) throws IOException {
+		SMALL_CONTENTS = Files.newInputStream(Paths.get(SMALL_CONTENTS_LOC)).readAllBytes();
+		LARGE_CONTENTS = Files.newInputStream(Paths.get(LARGE_CONTENTS_LOC)).readAllBytes();
```

Comment:
same here, should move to `getSpecsInSuite`

Reply:
Done.